Simcoe.podspec

```diff
@@ -34,7 +34,7 @@ Pod::Spec.new do |s|
   # Each subspec represents an analytics library implemented using Simcoe.
 
   adobe		  =   { :name => "Adobe",           :dependency => "AdobeMobileSDK", :version => '~> 4.13' }
```

Comment:
seems like the spacing is messed up here

Reply:
What should it be? I assumed it was intentional to vertically align the values

Reply:
i'm referring to this spacing `adobe		  =`
possibly it's just displaying oddly on GH?